server/datastore/mysql/users.go

```diff
@@ -172,9 +174,14 @@ func saveUserDB(ctx context.Context, tx sqlx.ExtContext, user *fleet.User) error
         sso_enabled = ?,
         mfa_enabled = ?,
         api_only = ?,
+				settings = ?,
```

Comment:
Nit: Weird identation.

Reply:
Ha, it's tabs v. spaces. My IDE normalizes them to look the same so I didn't notice.